storm-client/src/jvm/org/apache/storm/trident/topology/MasterBatchCoordinator.java

```diff
@@ -308,7 +308,7 @@ public String toString() {
                 ", _maxTransactionActive=" + _maxTransactionActive +
                 ", _coordinators=" + _coordinators +
                 ", _managedSpoutIds=" + _managedSpoutIds +
-                ", _spouts=" + _spouts +
+                ", spouts=" + _spouts +
```

Comment:
Do we want to get rid of the underscores of other variables? I wonder why only `spouts`

Reply:
This was probably the IDE getting overly ambitious when I refactored something else.  I'll fix it.